app/renderer/src/contexts/CounterContext.tsx

```diff
@@ -50,9 +50,9 @@ const CounterProvider: React.FC = ({ children }) => {
 
   const [shouldFullscreen, setShouldFullscreen] = useState(false);
 
-  const [count, setCount] = useState(config.stayFocus * 60);
```

Comment:
You can change the litteral translations but can you revert the variables. The issue is that these are stored as a json so it means the settings don't work/exist and result in an NAN for existing users
![image](https://github.com/zidoro/pomatez/assets/3209834/47d87327-38bd-4069-8bdb-741cab556295)

Reply:
![image](https://github.com/zidoro/pomatez/assets/3209834/85d89e7e-b02d-4cae-9549-0355417d83fd)